### PR TITLE
Base container for atlas-schemas

### PR DESCRIPTION
**Pull request by pcm32**
This PR aims to provide a base container for atlas-schemas and sketch a way of generating containers more systematically. Now from the correct branch.

### CODE REVIEW

File: .github/workflows/miniconda_templated.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Read tag
+      id: gettag
+      run: echo "::set-output name=tag::$(head -n 1 atlas-schemas-base/tag | awk -F':' '{ print $1 }' )"
+
+    - name: Read tag_version
+      id: gettagver
+      run: echo "::set-output name=tag_version::$(head -n 1 atlas-schemas-base/tag | awk -F':' '{ print $1 }' )"
```
Comment (pmb59) [inline .github/workflows/miniconda_templated.yaml:52]: Isn't `${{ steps.gettagver.outputs.tag_version }}` equal to `${{ steps.gettag.outputs.tag }}` ?

**Review by pmb59 — COMMENTED** [review]
LGTM. But I would recommend to double-check that the image (named `tag` here) and the tag (named `tag_version` here) follow the expected format for the action inputs, e.g:
```
registry: quay.io/my-namespace
image: my-image
tags: v1 v1.0.0
```
could this be clarified?

File: atlas-schemas-base/post_install_dockerfile [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+USER root
+RUN wget -qO- https://repo1.maven.org/maven2/org/flywaydb/flyway-commandline/6.3.2/flyway-commandline-6.3.2-linux-x64.tar.gz | tar xvz && ln -s `pwd`/flyway-6.3.2/flyway /usr/local/bin
```
[the remark this answers is not in the document]
Comment (pcm32) [reply atlas-schemas-base/post_install_dockerfile:3]: The outcome is the same apparently - I copied this from the currently working container at https://github.com/ebi-gene-expression-group/atlas-schemas/blob/master/Dockerfile - the idea is that that second container will just be based on this newer one.

**Review by pmb59 — APPROVED** [review]
Looks good to me - as long as you are happy with the nomenclature tag:tag_version

**Comment by pcm32** [comment]
Yes, maybe we should go with image and tag instead of tag and tag version.

**Comment by pcm32** [comment]
Tests are failing due to bioconda's index in anaconda being corrupted/down. We will need to wait here and try again.

**Pull request merged**